src/dataServices/apiDataService.ts

```diff
@@ -0,0 +1,116 @@
+/* eslint-disable class-methods-use-this */
+
+import {
+    ConditionalDeleteResourceRequest,
+    CreateResourceRequest,
+    DeleteResourceRequest,
+    GenericResponse,
+    PatchResourceRequest,
+    Persistence,
+    ReadResourceRequest,
+    ResourceNotFoundError,
+    UpdateResourceRequest,
+    vReadResourceRequest,
+} from 'fhir-works-on-aws-interface';
+import axios from 'axios';
+
+export class ApiDataService implements Persistence {
+    updateCreateSupported: boolean = false;
+
+    // TODO: Pull this value from AWS Param Store
+    static readonly INTEGRATION_TRANSFORM_URL = 'http://localhost:4000';
+
+    async createResource(request: CreateResourceRequest): Promise<GenericResponse> {
+        try {
+            const url = `${ApiDataService.INTEGRATION_TRANSFORM_URL}/${request.resourceType}`;
```

Comment:
These urls should be the SWAGGER api urls right?

Reply:
Yep, I'm using the API definitions here.
https://app.swaggerhub.com/apis/nguyen102/IntegrationFrameworkAPI/2#/

It doesn't include `persistence` in the route like the original API does, because from Integration Transform's point of view, I feel that including `persistence` is extraneous. 
https://app.swaggerhub.com/apis/nguyen102/IntegrationFrameworkAPI/2#/

Although, if Integration Transform does end up supporting search in the future, it would be convenient to have the resource of `persistence` or `search` added to the route. 

@bnurzhanov  what are your thoughts on using 
`POST [IntTran Base Route]/persistence/[resource-type]` to allow for future support of `GET [IntTran Base Route]/search`
vs
`POST [IntTran Base Route]/[resource-type]`

Reply:
Do we need to implement Integration Transform as RESTful API? The resource type is part of the payload and can be used to transform FHIR resource into payload formatted to comply with requirements of third-party system (for example as HL7v2 message).
I would vote for calling the `/` resource of the Integration Transform API endpoint

Reply:
I think to keep things consistent, it would be good to have it be RESTful. For example, when creating a resource we can do `POST /`. But to get a resource we would need to do `GET /Patient/[id]`. So instead of having two different resource path, we can keep it consistent by doing HTTP verbs on `/Patient`

Reply:
> `POST [IntTran Base Route]/persistence/[resource-type]` to allow for future support of `GET [IntTran Base Route]/search`
> vs
> `POST [IntTran Base Route]/[resource-type]`
Integration Transforms are stateless and therefore I think it is all just syntactic sugar that does not add additional information to the transform's execution. I am willing to disagree; and commit

Reply:
I see your point; and I agree.

Reply:
Changing route to include `persistence` in the route.

Reply:
@rsmayda  since we're now passing in the `INTEGRATION_TRANFORM_URL` into `apiDataService` from `deployment`. This change to use `persistence` is already taken care of.

Reply:
Updating route to include `/persistence` as a part of the path. The `deployment` package will pass in the base `IntegrationTransform` url and then each method in `apiDataService.ts` can append the proper resource to the base Url. For example `readResource` will append `/persistence` and `vReadResource` will append `/history`